modules/serverless-metering-datasources.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// serverless-metering.adoc
+
+[id="datasources-metering-serverless_{context}"]
+= Datasources for Knative Serving metering
+The following `ReportDataSources` are available for use with metering and the {ServerlessProductName} Knative Serving component.
```

Comment:
instead of "are available" maybe we should signify that these are examples of how it can be used with metering?